libs/mark-flow-ui/src/components/review.tsx

```diff
@@ -53,10 +53,15 @@ function CandidateContestResult({
   contest,
   vote = [],
   election,
+  isInterpretationResult,
 }: CandidateContestResultInterface): JSX.Element {
   const district = getContestDistrict(election, contest);
   const remainingChoices = contest.seats - vote.length;
 
+  const noVotesString = isInterpretationResult
+    ? appStrings.noteBallotContestNoSelection()
+    : appStrings.warningNoVotesForContest();
```

Comment:
Should we capitalize "no selection" for consistency?

Reply:
Yeah, SGTM